### PR TITLE
Update radio button style and example

### PR DESCRIPTION
**Pull request by lslachciak**
[Issue] https://github.com/Samsung/TAU/issues/735
[Problem] Radio button styles not up-to date
[Solution] Update radio button style. Update example

Signed-off-by: Lukasz Slachciak <l.slachciak@samsung.com>

### CODE REVIEW

File: src/css/profile/mobile/common/listview.less [hunk 1]
```diff
@@ -274,7 +274,10 @@ tau-expandable {
 			justify-content: flex-start;
 			margin-top: 33 * @unit_base;
 			margin-bottom: 33 * @unit_base;
-			padding: 0 32 * @unit_base;
+			padding-top: 0;
```
Comment (TomaszLukawskiSam) [inline src/css/profile/mobile/common/listview.less]: You set all padding directions. You can set simpler:
`padding: 0 24 * @px_base 0 24 * @px_base`

File: examples/mobile/UIComponents/components/controllers/radio.js [hunk 2]
```diff
@@ -5,18 +5,17 @@
...
 	/**
 	 * Updates text for an selected radio
 	 * @param {HTMLElement} target
 	 */
 	function setRadioresultFromTarget(target) {
-		if (target.checked) {
-			radioresult.innerHTML = "The Active Radio is " + target.id;
-		}
+		radiosTexts[target.id].innerHTML = "Radio on";
+		radiosTexts[(parseInt(target.id, 10) + 1) % 2].innerHTML = "Radio off";
```
Comment (TomaszLukawskiSam) [inline examples/mobile/UIComponents/components/controllers/radio.js]: You added `id` for  these elements. I think is better to use it. I think simpler is better, this is only sample.

**Review by TomaszLukawskiSam — CHANGES_REQUESTED** [review]

File: src/css/profile/mobile/common/listview.less (hunk 1 above)
Comment (lslachciak) [reply src/css/profile/mobile/common/listview.less]: done

File: examples/mobile/UIComponents/components/controllers/radio.js (hunk 2 above)
Comment (lslachciak) [reply examples/mobile/UIComponents/components/controllers/radio.js]: simplified

**Review by TomaszLukawskiSam — APPROVED** [review]
Looks good to me.

**Pull request merged**